pkg/eventemitter/eventemitter.go

```diff
@@ -0,0 +1,33 @@
+package eventemitter
+
+import (
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/names"
```

Comment:
please put self imports in a separate section:
```
import (
	"k8s.io/apimachinery/pkg/runtime"
	"k8s.io/client-go/tools/record"
	"sigs.k8s.io/controller-runtime/pkg/manager"

	"github.com/kubevirt/cluster-network-addons-operator/pkg/names"
```

Reply:
Done